bokeh/models/widgets/sliders.py

```diff
@@ -99,6 +102,9 @@ def __init__(self, **kwargs):
     * "mouseup": the callback will be executed only once when the slider is released.
 
     The "mouseup" policy is intended for scenarios in which the callback is expensive in time.
+
+    .. warning::
+    Callback policy currently apply to JS callbacks
```

Comment:
I believe this will need to be indented under `.. warning::` but compare to other instances to double check

Reply:
I check the proper indentation